lib/reek/context/code_context.rb

```diff
@@ -68,8 +68,9 @@ def initialize(_parent, exp)
       # @param type [Symbol] the type of the nodes we are looking for, e.g. :defs.
       # @yield block that is executed for every node.
       #
-      def local_nodes(type, &blk)
-        each_node(type, [:casgn, :class, :module], &blk)
+      def local_nodes(type, ignored = [], &blk)
+        ignored += [:casgn, :class, :module]
```

Comment:
Absolutely no need to apologise, learning and discussing is what pull requests are for :)